binocular:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  

  # Position Controllers ---------------------------------------

  joint0_position_controller:
    type: effort_controllers/JointPositionController
    joint: theta1
    pid: {p: 100.0, i: 0.01, d: 20.0}

  joint1_position_controller:
    type: effort_controllers/JointPositionController
    joint: theta2
    pid: {p: 100.0, i: 0.01, d: 20.0}

  joint2_position_controller:
    type: effort_controllers/JointPositionController
    joint: theta3
    pid: {p: 100.0, i: 0.01, d: 20.0}

  joint3_position_controller:
    type: effort_controllers/JointPositionController
    joint: theta4
    pid: {p: 100.0, i: 0.01, d: 20.0}

  joint4_position_controller:
    type: effort_controllers/JointPositionController
    joint: theta6
    pid: {p: 100.0, i: 0.01, d: 20.0}

  joint5_position_controller:
    type: effort_controllers/JointPositionController
    joint: theta7
    pid: {p: 100.0, i: 0.01, d: 20.0}